pkg/external_repos/community_repos.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "name": "EPEL 10 Everything x86_64",
+    "url": "https://dl.fedoraproject.org/pub/epel/10/Everything/x86_64/",
+    "distribution_version": "10",
+    "distribution_arch": "x86_64",
```

Comment:
i'm seeing the `distribution_arch` in the api responses for community repos show up as `any`. i think this field needs to be `arch` to map them properly. but i also think using `distribution_arch` everywhere would be good too and cause less confusion!

Reply:
oh also i think we'll need to import the epel repos for the playwright ui tests, do you mind adding a selector to each epel repo here so we can import them similarly to the small RH repo?